repos/fedora-updates.repo

```diff
@@ -0,0 +1,12 @@
+[updates]
```

Comment:
We have some weird issues with the `fedora-updates.repo` and `fedora.repo` files in the past. I wonder if with the new build system we can just remove these. They should already be in the base image. The only reason I can see to keep them around is in case we want to modify them to use a Beta release.